security/src/main/java/uk/ac/ox/ndph/mts/security/authentication/AADAuthenticationEntryPoint.java

```diff
@@ -5,6 +5,7 @@
 import org.springframework.security.web.AuthenticationEntryPoint;
 import org.springframework.web.bind.annotation.ControllerAdvice;
 import org.springframework.web.bind.annotation.ExceptionHandler;
+
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
```

Comment:
Could rename this file since it doesn't depend directly on anything AAD (uses Spring classes instead). Not significant.